demo-shell/src/app/components/file-view/file-view.component.html

```diff
@@ -76,20 +76,28 @@ <h1>My custom toolbar</h1>
         </adf-info-drawer-tab>
 
         <adf-info-drawer-tab label="Properties">
-            <adf-content-metadata-card *ngIf="isPreset" [node]="node"
-                                       [multi]="multi"
-                                       [preset]="customPreset"
-                                       [readOnly]="isReadOnly"
-                                       [displayAspect]="showAspect"
-                                       [displayDefaultProperties]="displayDefaultProperties"
-                                       [displayEmpty]="displayEmptyMetadata"></adf-content-metadata-card>
-
-            <adf-content-metadata-card *ngIf="!isPreset" [node]="node"
-                                       [multi]="multi"
-                                       [readOnly]="isReadOnly"
-                                       [displayAspect]="showAspect"
-                                       [displayDefaultProperties]="displayDefaultProperties"
-                                       [displayEmpty]="displayEmptyMetadata"></adf-content-metadata-card>
+            <adf-content-metadata
```

Comment:
This one tag is exactly the same like tag below. It has just different ngIf condition and inputs. 

Can we use this only once instead of duplicating and just use proper inputs values depending on case?

Reply:
good catch, removed the second condition